### DISCUSSION TITLE
How to call SetWindowText

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [microsoft/CsWin32#692] [Q&A] [answered]
I've added SetWindowTextA to my NativeMethods file, but it's not being picked up. I have...

```
using static Windows.Win32.PInvoke;
using Windows.Win32.UI.Input;
using Windows.Win32.Foundation;
using Windows.Win32.UI.WindowsAndMessaging;
```

I don't know what else to do.

**Comment by Gavin-Williams** [comment, marked as answer]
Got it .. use 'SetWindowText' instead of 'SetWindowTextA'.

**Reply by AArnott** [reply]
If you specifically want the narrow character version, you can specify `  "wideCharOnly": false` in the NativeMethods.json file and ask for `SetWindowTextA`.
